Missing file: /weblink/contents/biomes/catalog.json

### Expected behavior

N/A

### Observed/Actual behavior

It seems like https://api.github.com/repos/BentoBoxWorld/weblink/contents/biomes/catalog.json is missing?

```
[22:04:27 ERROR]: [BentoBox] An error occurred when downloading data from GitHub...
[22:04:27 ERROR]: [BentoBox] java.io.FileNotFoundException: https://api.github.com/repos/BentoBoxWorld/weblink/contents/biomes/catalog.json
        at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
        at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486)
        at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2070)
        at java.base/sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:2065)
        at java.base/java.security.AccessController.doPrivileged(AccessController.java:571)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:2064)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1634)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1614)
        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
        at BentoBox.jar//world.bentobox.bentobox.api.github.GitHubWebAPI.fetch(GitHubWebAPI.java:62)
        at BentoBox.jar//world.bentobox.bentobox.api.github.objects.repositories.GitHubFile.getContent(GitHubFile.java:29)
        at Biomes-2.2.0-1745213236927.jar//world.bentobox.biomes.web.WebManager.lambda$requestCatalogGitHubData$3(WebManager.java:86)
        at java.base/java.util.Optional.ifPresent(Optional.java:178)
        at Biomes-2.2.0-1745213236927.jar//world.bentobox.biomes.web.WebManager.requestCatalogGitHubData(WebManager.java:72)
        at Biomes-2.2.0-1745213236927.jar//world.bentobox.biomes.web.WebManager.lambda$new$0(WebManager.java:49)
        at org.bukkit.craftbukkit.scheduler.CraftTask.run(CraftTask.java:78)
        at org.bukkit.craftbukkit.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57)
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.io.FileNotFoundException: https://api.github.com/repos/BentoBoxWorld/weblink/contents/biomes/catalog.json
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:2009)
        at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1614)
        at java.base/java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:531)
        at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:307)
        at BentoBox.jar//world.bentobox.bentobox.api.github.GitHubWebAPI.fetch(GitHubWebAPI.java:57)
        ... 11 more


```

### Steps/models to reproduce

Load latest BentoBox with Biomes addon, exception would be thrown on startup.

### BentoBox version

> bbox ver
[23:22:42 INFO]: Running PAPER 1.21.4.
[23:22:42 INFO]: (1.21.4-226-a838a88 (MC: 1.21.4))
[23:22:42 INFO]: BentoBox version: 3.3.4-SNAPSHOT-b2877
[23:22:42 INFO]: Database: JSON
[23:22:42 INFO]: Loaded Game Worlds:
[23:22:42 INFO]: bskyblock_world (BSkyBlock): Overworld, Nether, The End
[23:22:42 INFO]: Loaded Addons:
[23:22:42 INFO]: Bank 1.9.0-SNAPSHOT-b113 (ENABLED)
[23:22:42 INFO]: Biomes 2.2.0 (ENABLED)
[23:22:42 INFO]: Border 4.5.0-SNAPSHOT-b242 (ENABLED)
[23:22:42 INFO]: BSkyBlock 1.19.1-SNAPSHOT-b793 (ENABLED)
[23:22:42 INFO]: Challenges 1.5.1 (ENABLED)
[23:22:42 INFO]: CheckMeOut 1.2.0 (ENABLED)
[23:22:42 INFO]: ControlPanel 1.15.0-SNAPSHOT-b86 (ENABLED)
[23:22:42 INFO]: DimensionalTrees 1.8.0 (ENABLED)
[23:22:42 INFO]: ExtraMobs 1.14.0 (ENABLED)
[23:22:42 INFO]: FarmersDance 1.2.2 (ENABLED)
[23:22:42 INFO]: IslandFly 1.14.2 (ENABLED)
[23:22:42 INFO]: Level 2.20.0-SNAPSHOT-b642 (ENABLED)
[23:22:42 INFO]: Likes 2.5.0-SNAPSHOT-b124 (ENABLED)
[23:22:42 INFO]: MagicCobblestoneGenerator 2.7.1-SNAPSHOT-b308 (ENABLED)
[23:22:42 INFO]: Visit 1.7.0-SNAPSHOT (ENABLED)
[23:22:42 INFO]: Warps 1.16.0-SNAPSHOT-b385 (ENABLED)
>


### Plugin list

_No response_

### Other

_No response_

--- a/src/main/java/world/bentobox/bentobox/api/github/objects/repositories/GitHubFile.java
+++ b/src/main/java/world/bentobox/bentobox/api/github/objects/repositories/GitHubFile.java
@@ -21,12 +21,17 @@ public class GitHubFile {
 
     /**
      * Returns the content of this file.
-     * @return the content of this file in Base64
-     * @throws IllegalAccessException if the connection to the GitHub API could not be established.
+     * @return the content of this file in Base64 or nothing if the connection to the GitHub API could not be established.
      * @throws IOException - If an error occurs during the request.
      */
-    public String getContent() throws IllegalAccessException, IOException {
-        JsonObject response = api.fetch(path);
+    public String getContent() throws IllegalAccessException {
+        JsonObject response;
+        try {
+            response = api.fetch(path);
+        } catch (IOException e) {
+            // Cannot get a connection for some reason
+            return "";
+        }
         return response.get("content").getAsString();
     }
 }

--- a/src/test/java/world/bentobox/bentobox/util/ItemParserTest.java
+++ b/src/test/java/world/bentobox/bentobox/util/ItemParserTest.java
@@ -143,6 +143,12 @@ public class ItemParserTest {
             // TODO Auto-generated method stub
             return 0;
         }
+
+        @Override
+        public Stream keyStream() {
+            // TODO Auto-generated method stub
+            return null;
+        }
     }
 
     @After

